lib/protoflow-blocks/src/blocks/sys/write_socket.rs

```diff
@@ -0,0 +1,185 @@
+extern crate std;
```

Comment:
Insert a newline prior after `extern crate` (and run `cargo fmt`!)

Reply:
done